plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/LocatedTableHandle.java

```diff
@@ -24,4 +25,6 @@ public interface LocatedTableHandle
     boolean managed();
 
     String location();
+
+    VendedCredentialsHandle toCredentialsHandle();
```

Comment:
Do we still need the above managed() and location()? VendedCredentialsHandle contains those information in my understanding.

Reply:
we are not holding the instance of `VendedCredentialsHandle` in `LocatedTableHandle` sub classes, or
do you mean change the method to default? i,e `boolean managed()` to 
```
default boolean managed() 
{ 
    return toCredentialsHandle().managed();
}
```